README.md

```diff
@@ -307,6 +307,23 @@ messages.after-failure = ""
 # A message to print after a successful build.
 messages.after-success = ""
 
+# Add the wheel install path to the CMake prefix paths.
+search.use-install-prefix = true
+
+# Add the wheel build path to the CMake prefix paths.
+search.use-build-prefix = true
```

Comment:
That's not what these paths are. It's just trying to find site-packages, and sometimes the wrong path is reported (I think possibly if there's a user site-packages?), so we also make sure scikit-build-core's parent directory is included. I think this should be just one setting: `search.use-site-packages`.